src/kibana/plugins/vis_types/controls/vislib_basic_options.html

```diff
@@ -17,4 +17,28 @@
       Scale Y-Axis to Data Bounds
     </label>
   </div>
+  <div class="vis-option-item">
+    <label>
+      <input type="checkbox" ng-model="vis.params.setYExtents.max" ng-checked="vis.params.setYExtents.max">
+      Set Y-Axis Max Value
+    </label>
+    <div ng-show="vis.params.setYExtents.max">
+      <label>
+        y-max
+        <input type="number" step="0.1" ng-model="vis.params.yAxis.max" ng-required="vis.params.setYExtents.max">
```

Comment:
You could use min here to make sure people can't input numbers < 0.

Just a suggestion since I see [this](https://github.com/elastic/kibana/pull/3464/files#diff-6c81d70d133647828c422c618852eb30R111)